protos/rust/src/pb/aptos.util.timestamp.rs

```diff
@@ -2,8 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // @generated
-// This file is @generated by prost-build.
-#[derive(Clone, Copy, PartialEq, ::prost::Message)]
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
```

Comment:
This shouldn't change, can you check your prost version?

Reply:
prost = { version = "0.13.4", features = ["no-recursion-limit"] }
prost-types = "0.13.3"